shared/js/content-scripts/onboarding.js

```diff
@@ -0,0 +1,12 @@
+(function () {
+    // Note: this is only used on duckduckgo.com to assist with user onboarding
+    if (document.readyState === 'interactive' || document.readyState === 'complete') {
+        start()
+    } else {
+        document.addEventListener('DOMContentLoaded', start)
+    }
+
+    function start () {
+        window.postMessage({ type: 'onboarding', ready: true }, '*')
```

Comment:
According to MDN seems like we should really try to avoid using `*` for the target origin.

I'm thinking we should have `onboarding.js` and `onboarding-dev.js` where the target origin is `'https://duckduckgo.com'` and `'*'`.

We can add a `setDevMode` method to ATB.js and enable it in [debug.es6.js](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/background/debug.es6.js). With that we can load either the production or dev file accordingly.